pkgs/development/python-modules/matrixzulipbridge/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+}:
+buildPythonPackage rec {
+  pname = "MatrixZulipBridge";
+  version = "0.4.1";
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = lib.fakeSha256;
```

Comment:
what is the real hash here?

Reply:
I did not obtain it by now. Sadly, I have no idea how to test this, could you help me out here?

(`nix-build -A matrixzulipbridge` did not work)

Reply:
TL;dr Run `nix-build -A matrixzullipbridge` and find the `hash mismatch` error. It will give you the proper hash value.

Details: https://ryantm.github.io/nixpkgs/builders/fetchers/#chap-pkgs-fetchers-caveats

P.S. There are ways of asking for the hashes of many things from the command line, but this quick and dirty method works well.

Reply:
P.S. Replace `lib.fakeSha256` with an empty string if `nix-build` isn't building anything.

Reply:
```bash
➜  MatrixZulipBridge git:(package/MatrixZulipBridge) nix-build -A matrixzulipbridge
error: attribute 'matrixzulipbridge' in selection path 'matrixzulipbridge' not found
```

It does not work :( is there anything that I need to add additionally to the `package.nix` and `python-packages.nix`?

Reply:
Use `nix-build -A python312Packages.matrix-zulip-bridge` (replace 312 with whichever version you're using)

Reply:
Thank you :D finally, I got it working!

Just wanted to say thank you, I wouldn't have gotten it to work all by myself :D ❤️

Reply:
You're welcome! Happy to help. (The first time is always a learning experience.)